### PR TITLE
Bugowner tool

### PR DESCRIPTION
**Pull request by lnussel**

### CODE REVIEW

File: bugowner.py [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+        for p in persons:
+            o = Owner('person', p)
+            logger.info("%s -> %s %s", package, o.kind, o.name)
+            self.tool.add_bugowner(p, o)
+
+    @cmdln.option('--set', action='store_true',
+                  help='request bugowner')
+    @cmdln.option('--request', action='store_true', help='print osc request lines')
+    def do_lastsubmitter(self, subcmd, opts, *packages):
+        """${cmd_name}: show last comitter for packages
```
Comment (jberry-suse) [inline bugowner.py]: s/comitter/committer/

**Comment by lnussel** [comment]
pr was update, @jberry-suse ok now?

**Comment by jberry-suse** [comment]
Tests missing dep.

**Comment by jberry-suse** [comment]
Also, `bugowner.py` should be executable and included in `.spec` bin list.

**Pull request merged**